### PR TITLE
SQSCANGHA-29 Update .scannerwork cleanup

### PR DESCRIPTION
**Pull request by istvan-bodo-sonarsource**

### CODE REVIEW

**Review by antoine-vinot-sonarsource — APPROVED** [review]
LGTM

File: cleanup.sh [hunk 1]
```diff
@@ -2,7 +2,12 @@
 
 set -e
 
+if [ ! -d "${INPUT_PROJECTBASEDIR}/.scannerwork" ]; then
```
Comment (henryju) [inline cleanup.sh:5]:
```suggestion
if [ ! -d "${INPUT_PROJECTBASEDIR%/}/.scannerwork" ]; then
```

File: cleanup.sh [hunk 2]
```diff
@@ -2,7 +2,12 @@
...
+    echo ".scannerwork directory not found; nothing to clean up."
+    exit
+fi
+
 _tmp_file=$(ls "${INPUT_PROJECTBASEDIR}/" | head -1)
 PERM=$(stat -c "%u:%g" "${INPUT_PROJECTBASEDIR}/$_tmp_file")
 
-chown -R $PERM "${INPUT_PROJECTBASEDIR}/"
\ No newline at end of file
+chown -R $PERM "${INPUT_PROJECTBASEDIR}/.scannerwork/"
```
Comment (henryju) [inline cleanup.sh:13]:
```suggestion
chown -R $PERM "${INPUT_PROJECTBASEDIR%/}/.scannerwork/"
```

File: cleanup.sh [hunk 3]
```diff
@@ -2,7 +2,12 @@
 
 set -e
 
+if [ ! -d "${INPUT_PROJECTBASEDIR}/.scannerwork" ]; then
+    echo ".scannerwork directory not found; nothing to clean up."
+    exit
+fi
+
 _tmp_file=$(ls "${INPUT_PROJECTBASEDIR}/" | head -1)
```
Comment (henryju) [inline cleanup.sh:10]:
```suggestion
_tmp_file=$(ls "${INPUT_PROJECTBASEDIR%/}/" | head -1)
```

File: cleanup.sh [hunk 4]
```diff
@@ -2,7 +2,12 @@
...
 PERM=$(stat -c "%u:%g" "${INPUT_PROJECTBASEDIR}/$_tmp_file")
```
Comment (henryju) [inline cleanup.sh:11]:
```suggestion
PERM=$(stat -c "%u:%g" "${INPUT_PROJECTBASEDIR%/}/$_tmp_file")
```

**Review by henryju — APPROVED** [review]
Minor suggestions, in case the users have a trailing slash

**Review by henryju — APPROVED** [review]

**Pull request merged**